docs/source/gateway/index.rst

```diff
@@ -1,6 +1,782 @@
+
+      - name: embeddings
+        type: llm/v1/embeddings
+        model:
+          provider: openai
+          name: text-embedding-ada-002
+          config:
+            openai_api_base: https://api.openai.com/v1
+            openai_api_key: $OPENAI_API_KEY
+
+Save this file to a location on the system that is going to be running the MLflow AI Gateway server.
+
+Step 4: Start the Gateway Service
+---------------------------------
+You're now ready to start the Gateway service!
+
+Use the MLflow AI Gateway ``start`` command and specify the path to your configuration file:
+
+.. code-block:: sh
+
+    mlflow gateway start --config-path config.yaml --port {port} --host {host} --workers {worker count}
+
+If you do not specify the host, a localhost address will be used.
+
+If you do not specify the port, port 5000 will be used.
+
+The worker count for gunicorn defaults to 2 workers.
+
+Step 5: Access the Interactive API Documentation
+------------------------------------------------
+The MLflow AI Gateway service provides an interactive API documentation endpoint that you can use to explore
+and test the exposed routes. Navigate to ``http://{host}:{port}/`` (or ``http://{host}:{port}/docs``) in your browser to access it.
+
+The docs endpoint allow for direct interaction with the routes and permits submitting actual requests to the
+provider services by click on the "try it now" option within the endpoint definition entry.
+
+Step 6: Send Requests to Routes via REST API
+--------------------------------------------
+You can now send requests to the exposed routes.
+See the :ref:`REST examples <gateway_rest_api>` for guidance on request formatting.
+
+Step 7: Send Requests Using the Fluent API
+------------------------------------------
+
+Here's an example of how to send a chat request using the ``fluent`` API:
+
+.. code-block:: python
+
+    from mlflow.gateway import query, set_gateway_uri
+
+    set_gateway_uri(gateway_uri="http://localhost:5000")
+
+    response = query(
+        "chat",
+        {"messages": [{"role": "user", "content": "What is the best day of the week?"}]},
+    )
+
+    print(response)
+
+**Note:** Remember to change the uri definition to the actual uri of your Gateway server.
+
+The returned response will be in this data structure (the actual content and token values will likely be different):
+
+.. code-block:: python
+
+    {
+        "candidates": [
+            {
+                "message": {
+                    "role": "assistant",
+                    "content": "\n\nIt's hard to say what the best day of the week is.",
+                },
+                "metadata": {"finish_reason": "stop"},
+            }
+        ],
+        "metadata": {
+            "input_tokens": 13,
+            "output_tokens": 15,
+            "total_tokens": 28,
+            "model": "gpt-3.5-turbo-0301",
+            "route_type": "llm/v1/chat",
+        },
+    }
+
+
+Step 8: Send Requests Using the Client API
```

Comment:
Can we add a Step 9: Try out different models and show an example of how to add another model from another provider and then link to the gateway configuration section & gateway additional configuration details section?